tests/ok_ko.ml

```diff
@@ -18,7 +18,10 @@ let _ =
   | None -> assert false
   | Some(f) -> Package.apply_config f;
   let open Alcotest in
-  let files = Sys.readdir "OK" |> Array.map (fun f -> "OK/" ^ f) in
+  let files = Sys.readdir "OK" |> Array.map (fun f -> "OK/" ^ f)
+  |> Array.to_list
+  |> List.filter (function f -> f <> "OK/unif_hint.lp")
+  |> Array.of_list in
```

Comment:
thanks to add a comment saying that this is temporary
thanks to use fun instead of function everywhere, for uniformity

Reply:
fixed

Reply:
why github doesn't write yellow circled Outdated ?